src/engine/Engine.ts

```diff
@@ -754,8 +788,80 @@ O|===|* >________________>\n\
             this._logger.info('parent.clientHeight ' + parent.clientHeight);
             this.setAntialiasing(this._isSmoothingEnabled);
          });
+      } else if (this.displayMode === DisplayMode.Position) {
+          
+          if ( !options.position ) {
+            throw new Error('DisplayMode of Position was selected but no position option was given');
+          } else {
+              
+              this.canvas.style.display = 'block';
+              this.canvas.style.position = 'absolute';
+              
+              if (typeof options.position === 'string') {
+                var specifiedPosition = options.position.split(' ');
+                
+                switch (specifiedPosition[0]) {
+                  case 'top':
+                    this.canvas.style.top = '1%';
```

Comment:
No particular reason, I just personally though having a little border around the edge looked better :) I've set it to 0px now!